dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -573,6 +573,21 @@ def type_into_selector(input_text, selector)
   }
 end
 
+Then /^I reopen the congrats dialog unless I see the sharing input/ do
+  next if @browser.execute_script("return $('#sharing-input').length > 0;")
+  puts "reopening congrats dialog"
```

Comment:
I do want to keep this puts around, so that we can see if this codepath was executed when we look at the cucumber logs.